datahub-web-react/src/app/entity/shared/Ownership.tsx

```diff
@@ -105,13 +105,14 @@ export const Ownership: React.FC<Props> = ({ owners, lastModifiedAt, updateOwner
         setStagedOwners(newStagedOwners);
     };
 
-    const onDelete = (urn: string) => {
+    const onDelete = (urn: string, type: OwnershipType) => {
         const updatedOwners = owners
-            .filter((owner) => !(owner.owner.urn === urn))
+            .filter((owner) => !(owner.owner.urn === urn && owner.type === type))
```

Comment:
is it possible for an two owners to have the same urn but not have the same type?

Reply:
Sorry, i think i need to filter it with role and not with type
Right ?

Reply:
Because a user can have multiple role there

Reply:
So it should be ``` owner.type === role ```

Reply:
Ah I see- I think that may be the solution.